packages/core/admin/admin/src/content-manager/pages/ListView/components/TableRows/index.js

```diff
@@ -53,7 +59,9 @@ export const TableRows = ({
   const handleCloneClick = (id) => async () => {
     try {
       const { data } = await post(
-        `/content-manager/collection-types/${contentType.uid}/auto-clone/${id}?${pluginsQueryParams}`
+        `/content-manager/collection-types/${contentType.uid}/auto-clone/${id}`,
+        {},
+        { params: { plugins: query?.plugins } || {} }
```

Comment:
won't this object always be true? is the `||` in the wrong place?

Reply:
Yeah thanks it was in the wrong place, but I think the check is unnecessary